htdocs/install/mysql/migration/5.0.0-6.0.0.sql

```diff
@@ -395,3 +395,37 @@ ALTER TABLE llx_usergroup_rights DROP INDEX fk_usergroup;
 ALTER TABLE llx_usergroup_rights ADD UNIQUE INDEX uk_usergroup_rights (entity, fk_usergroup, fk_id);
 ALTER TABLE llx_usergroup_rights ADD CONSTRAINT fk_usergroup_rights_fk_usergroup FOREIGN KEY (fk_usergroup) REFERENCES llx_usergroup (rowid);
 
+CREATE TABLE llx_blockedlog 
+( 
+	rowid integer AUTO_INCREMENT PRIMARY KEY, 
+	tms	timestamp DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+	action varchar(50), 
+	amounts real NOT NULL, 
+	signature varchar(100) NOT NULL, 
+	signature_line varchar(100) NOT NULL, 
+	element varchar(50), 
+	fk_object integer,
+	ref_object varchar(100), 
+	date_object	datetime,
+	object_data	text,
```

Comment:
Let's keep 2 fields only for the moment. We will see later if we need distinct column with experience.